ledger/participant-state/protobuf/ledger_configuration.proto

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
```

Comment:
@dajmaki why did you put the configuration proto file at this location and not under kvutils?
![image](https://user-images.githubusercontent.com/25179017/68160044-ee168380-ff20-11e9-9931-f1d0850e1416.png)

Reply:
yes, this would be used by all implementations and the SQL index. But I realized that we need to make sure all fields make sense for all ledger implementations, and some of the things don't actually work that well (authorized participant id and open world). We'll need to rethink this a bit.